.circleci/config.yml

```diff
@@ -240,13 +254,13 @@ jobs:
           env: "staging"
       - notify-qa:
           channel: "C7H40L71D" # dsde-qa-notify
-  integration-tests-branch-against-dev:
+  integration-tests-branch-against-staging:
     executor: puppeteer
     parallelism: 4
     steps:
       - integration-tests:
           local: true
-          env: local
+          env: staging
```

Comment:
The env here should stay as `local`. Changing it to `staging` runs tests against the UI deployed on staging, not the local UI in the branch.

Reply:
You're right. Purpose of `local` and `env` were a bit confusing for me.